packages/client/src/client.ts

```diff
@@ -49,7 +49,7 @@ export type SignInOptions = {
   postRedirectUri?: string | URL;
 } & Pick<
   SignInUriParameters,
-  'interactionMode' | 'firstScreen' | 'loginHint' | 'directSignIn' | 'extraParams'
+  'interactionMode' | 'firstScreen' | 'loginHint' | 'directSignIn' | 'extraParams' | 'prompt'
```

Comment:
Should we add some comment to indicate the prompt here will override the config one?

Reply:
ok